pkg/volume/util/subpath/subpath_linux_test.go

```diff
@@ -539,6 +540,45 @@ func TestCleanSubPaths(t *testing.T) {
 				return validateDirExists(baseSubdir)
 			},
 		},
+		{
+			name: "subpath-with-files",
+			prepare: func(base string) ([]mount.MountPoint, error) {
+				path := filepath.Join(base, containerSubPathDirectoryName, testVol, "container1", "0")
+				path2 := filepath.Join(base, containerSubPathDirectoryName, testVol, "container1", "1")
```

Comment:
It's also possible for the path to be a file. Can you add a test case for that?

Reply:
I was about to add a case for subpath with file, but turns out it already have one test case for that

https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/subpath/subpath_linux_test.go#L434

Do you see something else to cover?